modal/sandbox.py

```diff
@@ -567,6 +567,21 @@ async def open(
         task_id = await self._get_task_id()
         return await _FileIO.create(path, mode, self._client, task_id)
 
+    async def ls(self, path: str) -> list[str]:
```

Comment:
j/w but did you consider using `Path` objects here instead of `str`?

Reply:
I hadn't, but it seems like it would be a good addition to support it. There are a few other instances in `FileIO` that take in `str` paths only